### DISCUSSION TITLE
Where to find settings / config files on linux

### DISCUSSION THREAD

**Discussion by Andor14** [jeffvli/feishin#689] [Q&A]
Hi, where can i find the saved settings / config files on linux? Where things like font, theme, accent color and hotkeys are found.

**Comment by jeffvli** [comment]
They're saved in your browser's localStorage under `store_settings`.

**Reply by Anexgohan** [reply]
may i ask, where are they stored when running through docker compose?

**Reply by kgarner7** [reply]
In your web browser. Docker is just for serving the webpage
